icons/book-elastic.svg

```diff
@@ -0,0 +1,14 @@
+<svg
```

Comment:
Not sure about the use case this one.

Reply:
@ericfennis Ah I missed this one off the list above https://github.com/lucide-icons/lucide/pull/1352#issue-1748983530

It’s a (closed) note/sketchbook/diary/Moleskine… The elastic strap makes it distinct from `book`.

Reply:
I understand what it is, but what is the use-case of it? Like where can we use it for?
I'm doubting if one line will make a real difference in sending the message that it is a note/sketchbook or diary. I think the book will do the same job, and book-heart is probably a better fit for a diary use case. 
A real use case will maybe a webshop selling office stuff, but I think the design of the book and this one is too similar for that use case.